versions/2023-11-15/privacyPolicy.md

```diff
@@ -1,6 +1,6 @@
 # Terra Platform Privacy Policy
 
-Last Updated: November 15, 2023
+Last Updated: October 17, 2025
```

Comment:
I wasn't sure when this would merge and then get deployed to prod.  It's apparently just static but presumably will be picked up whenever Sam restarts, so if this gets merged before tomorrow's deployment then should the date be the 16th?